### ISSUE TITLE
About part name "cherry light"

### ISSUE THREAD

**Issue by Massetstephane** [KSP-KOS/KOS#2896]
Hi !
I'm not sure but for name in part cfg file space may be a problem. (folder : KOSCherryLight) 

>from wiki :  **name** -> set the hidden name of the part. Spaces are not allowed.

I wanted to make myself a small personal patch to add a scale to this light, and the space in the name is not good it would take "CherryLight"

steph.

**Comment by Dunbaratu** [comment]
Can you link to that wiki page you're referencing so I can read what you mean?

**Comment by JonnyOThan** [comment]
Seems like it's from here: https://wiki.kerbalspaceprogram.com/wiki/CFG_File_Documentation#General_Parameters

However I think you can target parts with spaces in MM patches if you use a . or ? character.  And renaming an already-existing part will break peoples' save games - you'd have to make a clone of it with the changed name and hide the original.

**Comment by nuggreat** [comment]
I have to wonder if that was something that was true at one point and got changed or if it was assumed to be true and no one questioned it.  Because as it stands having the space in there doesn't appear to cause any issues with how the part works baring other mods doings stuff based on the assumption that there can be no spaces and that causing an issue.

**Comment by JonnyOThan** [comment]
Yeah, this part certainly isn't true considering squad has underscores in some of their part names:

> Underscores and other special characters are either not allowed or not recommended.

Spaces *do* cause a little hassle for MM patches, but there is a workaround as I mentioned.

**Comment by nuggreat** [comment]
For a personal one off patch it would likley be faster and easier to just add the module to the part file your self as apposed to mucking about with MM configs.  If kOS updated more frequently then I would recommend the MM patch but as it doesn't just editing the config would likely work well enough.

**Comment by Massetstephane** [comment]
Ok thank you, yes I had taken the info [here](https://wiki.kerbalspaceprogram.com/wiki/CFG_File_Documentation#INTERNAL).
Steph
